modules/react/breadcrumbs/lib/hooks/useBreadcrumbsModel.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {createModelHook} from '@workday/canvas-kit-react/common';
+import {defaultGetId, useOverflowListModel} from '@workday/canvas-kit-react/collection';
+import {useMenuModel} from '@workday/canvas-kit-react/menu';
+import {reorganizeHiddenItems} from './utils';
+
+export const useBreadcrumbsModel = createModelHook({
+  defaultConfig: {
+    ...useOverflowListModel.defaultConfig,
+    /**
+     * Optional id for the whole `ActionBar` group. If not provided, a unique id will be created.
```

Comment:
I assume this is not ActionBar?

Reply:
Thanks for catching it! I forgot to modify this file after updating functionality!

Reply:
All good!